solutions/logging/client-env/folder-sink.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+######
+# Folder sink for Platform and Component logs of Client Resources
+# Destination: cloud logging bucket inside logging project
+apiVersion: logging.cnrm.cloud.google.com/v1beta1
+kind: LoggingLogSink
+metadata:
+  name: platform-and-component-log-client1-log-sink # kpt-set: platform-and-component-log-${client-name}-log-sink
+  namespace: logging
+  annotations:
+    config.kubernetes.io/depends-on: logging.cnrm.cloud.google.com/namespaces/logging/LoggingLogBucket/platform-and-component-client1-log-bucket # kpt-set: logging.cnrm.cloud.google.com/namespaces/logging/LoggingLogBucket/platform-and-component-${client-name}-log-bucket
+    internal.kpt.dev/upstream-identifier: 'logging.cnrm.cloud.google.com|LoggingLogSink|logging|platform-and-component-log-client1-log-sink'
```

Comment:
the `internal.kpt.dev/upstream-identifier:` annotation should be removed

Reply:
removed